client/src/components/InterpreterSignUpForm.js

```diff
@@ -0,0 +1,171 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { Button, Form, Grid, Select, Header } from "semantic-ui-react";
+import languageOptions from "./LanguageOptions";
+
+class InterpreterSignUpForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      username: "",
+      password: "",
+      confirmationpassword: "",
+      languages: [],
+      userCreated: false
+    };
+  }
+  handleChange = (event, { value, name }) => {
+    this.setState({
+      [name]: value
+    });
+    console.log(value);
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const {
+      password,
+      confirmationpassword,
+      email,
+      username,
+      languages
+    } = this.state;
+    // perform all neccassary validations
+    if (password !== confirmationpassword) {
+      alert("Passwords don't match");
+    } else {
+      // make API call
+      const interpreterRequest = {
+        method: "POST",
+        body: JSON.stringify({
+          email,
+          username,
+          password,
+          confirmationpassword,
+          languages
+        }),
+        headers: {
+          "Content-Type": "application/json"
+        }
+      };
+      fetch("http://localhost:4000/auth/register", interpreterRequest)
+        .then(res => res.json())
+        .then(this.resetForm);
+    }
+  };
+  resetForm = () => {
+    this.setState({
+      email: "",
+      username: "",
+      password: "",
+      confirmationpassword: "",
+      languages: [],
+      userCreated: true
+    });
+  };
+  handleClick = () => {
+    window.location.href = "/";
+  };
+  render() {
+    return (
+      <div>
+        {this.state.userCreated ? (
+          <p> succeccful</p>
```

Comment:
You can use this component to display a confirmation message: https://react.semantic-ui.com/collections/message/#variations-success. And write something like: "Your account has been successfully created." This message can be between the title and the form.